contextproject-ms3/src/main/java/contextproject/models/Track.java

```diff
@@ -263,6 +263,62 @@ public void energyLevels() {
   }
 
   /**
+   * Write average energy, intro and outro transitions to id3 tag.
+   */
+  public void id3EnergyWiriter() {
+    try {
+      String res = averageEnergy + "-" + inTransitionTimes.toString() + "-"
+          + outTransitionTimes.toString();
+
+      tag.deleteField("TXXX");
```

Comment:
Since this tag is used several times, maybe we can put it in a final variable? (not hardcode it into the methods)